f/php/template.yaml

```diff
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: 'Bref bootstrap benchmark F - PHP'
+
+Globals:
+    Function:
+        Timeout: 10
+        MemorySize: 1024
+
+Resources:
+    DemoFunction:
+        Type: AWS::Serverless::Function
+        Properties:
+            FunctionName: 'bref-bootstrap-benchmark-f-php'
+            Description: 'Bref bootstrap benchmark F - PHP'
+            CodeUri: .
+            Handler: bref.php
```

Comment:
Does this need to be index.php? Or is this ignored?

Reply:
This is ignored for now. I think AWS puts that in an environment variable. For the benchmark it doesn't really matter.